src/main/java/org/gridsuite/modification/server/modifications/GenerationDispatch.java

```diff
@@ -522,13 +523,65 @@ public void apply(Network network, Reporter subReporter) {
             Reporter resultReporter = componentReporter.createSubReporter(RESULT, RESULT);
 
             if (Math.abs(totalAmountSupplyToBeDispatched - realized) < EPSILON) {
+                Map<String, List<Generator>> generatorsByRegion = getGeneratorsByRegion(network);
+
                 report(resultReporter, Integer.toString(componentNum), "SupplyDemandBalanceCouldBeMet", "The supply-demand balance could be met",
                     Map.of(), TypedValue.INFO_SEVERITY);
+                generatorsByRegion.forEach((region, generators) -> report(resultReporter, Integer.toString(componentNum), "SumGeneratorActivePower" + region, "Sum of generator active power setpoints in ${region} region: ${sum} MW.",
+                        Map.of("region", region, "sum", getSumActivePower(generators)), TypedValue.INFO_SEVERITY));
             } else {
                 double remainingPowerImbalance = totalAmountSupplyToBeDispatched - realized;
                 report(resultReporter, Integer.toString(componentNum), "SupplyDemandBalanceCouldNotBeMet", "The supply-demand balance could not be met : the remaining power imbalance is ${remainingPower} MW",
                     Map.of("remainingPower", remainingPowerImbalance), TypedValue.WARN_SEVERITY);
             }
         }
     }
+
+    private Map<String, List<Generator>> getGeneratorsByRegion(Network network) {
```

Comment:
You should add also the synchronous component to this method arguments, and filter the generators that are inside this synchronous component (see computeTotalAmountFixedSupply method for example)

Reply:
Done.